mod_reforged/hooks/entity/tactical/humans/swordmaster.nut

```diff
@@ -54,18 +54,51 @@
 	{
 		if (this.m.Items.hasEmptySlot(::Const.ItemSlot.Body))
 		{
+			local armor = null;
 			if (this.m.MyArmorVariant == 0) // light armor
 			{
-				this.m.Items.equip(::new("scripts/items/armor/noble_mail_armor"));
+				armor = ::new("scripts/items/armor/noble_mail_armor");
```

Comment:
This needs to be just the script path, not an instantiated armor with `::new` because you are instantiating it on line 72 below. That is why you get the error because on that line it tries to instantiate an already instantiated object instead of instantiating a script.